.vscode/launch.json

```diff
@@ -96,10 +96,14 @@
       "port": 9091,
       "cwd": "${workspaceFolder}",
       "envFile": "${workspaceFolder}/.flaskenv",
-      "env": { "PYTHONPATH":"${workspaceFolder}"},

```

Comment:
We are still going to be able to do this for COF - could you comment out the COF stuff instead of remove and add something to the README about configuring launch.json for different funds/rounds?